src/scenic/syntax/compiler.py

```diff
@@ -9,10 +9,16 @@
 def compileScenicAST(scenicAST: ast.AST) -> Tuple[ast.AST, List[ast.AST]]:
     """Compiles Scenic AST to Python AST"""
     compiler = ScenicToPythonTransformer()
-    node = ast.fix_missing_locations(compiler.visit(scenicAST))
+    node = compiler.visit(scenicAST)
+    # call `fix_missing_locations` if node is an AST node
+    if isinstance(node, ast.AST):
```

Comment:
`node` can be a list in tests and `fix_missing_locations` cannot be called. We skip this if `node` is not an AST node.